06_gpu_and_ml/nerf_studio.py

```diff
@@ -0,0 +1,73 @@
+import modal
+import subprocess  # Import subprocess to fix the NameError
+
+gpu_device = "a100"
+
+# Define the container image using Conda for package management
+image = (
+    modal.Image.conda()
+    .apt_install("git")
+    .run_commands(
+        "apt-get update",
+        "apt-get install -y wget build-essential git",
+        "pip install --upgrade pip",
+    )
+    .conda_install(
+        "nvidia::cuda-toolkit=12.4.1",
```

Comment:
I'm kinda surprised this works? The host driver is 12.2.

Is a later step maybe installing the CUDA toolkit at <=12.2?